recipe/meta.yaml

```diff
@@ -13,7 +13,7 @@ requirements:
     - dask =2022.5.0
     - distributed =2022.5.0
     - dask-kubernetes =2022.4.1
-    - dask-gateway =0.9.0
+    - dask-gateway =2022.4.0
```

Comment:
It's awesome to see a new release, thanks @consideratio!! For pangeo hubs this will have breaking changes it seems https://github.com/dask/dask-gateway/blob/main/docs/source/changelog.md#202240 , so either need to pin this or update existing hubs and binderhubs.

Reply:
@scottyhq sadly there is an issue it seems - and this is why this PRs CI tests are failing.

I don't get it yet myself, if you have ideas, please chime in at https://github.com/conda-forge/dask-gateway-feedstock/issues/29.

Reply:
Ohhh, wait it seems like there must have been some cache or similar. Now things seem to work.

Can you re-run this CI check?